test/base/test_engine.py

```diff
@@ -19,7 +19,8 @@ def test_basic():
     for engine in [
         pypesto.engine.SingleCoreEngine(),
         pypesto.engine.MultiProcessEngine(n_procs=2),
-        pypesto.engine.MultiThreadEngine(n_threads=8),
+        pypesto.engine.MultiProcessEngine(n_procs=2, method="spawn"),
+        pypesto.engine.MultiThreadEngine(n_threads=4),
```

Comment:
Why is only "spawn" additionally to default tested? shouldn't it be at least one more?